docs/lake/design.md

```diff
@@ -588,7 +606,7 @@ in a pool, e.g.,
 ```
 zed lake index apply -p logs IndexGroupEx <tag>
 ```
-The index is created and a transaction put in staging.  Once this transaction
+The index is created and a transaction put (somewhere).  Once this transaction
```

Comment:
This one needs genuine input! I just put some garbage text in there to trigger the PR suggestion workflow.

I know that "staging" as a formal construct is gone, but I don't know what took its place in this context. I know indexing in general is not operational (#2532), but I figure if we're going to make forward-looking statements about how it's expected to work, it might as well be accurate. :smile:

Reply:
The index is created and transactionally committed to the branch...